src/containers/Classroom.jsx

```diff
@@ -10,19 +10,21 @@ import { deleteClassroom, deleteStudent } from '../actions/teacher';
 export default class Classroom extends Component {
 
   render() {
-    const { classrooms, params, actions } = this.props;
+    const { classrooms, assignments, params, actions } = this.props;
     const members = classrooms.members;
     const classroom = classrooms.data.find(classroom => classroom.id === params.classroomId);
     const classroomId = (classroom && classroom.id) ? classroom.id : undefined;
     const boundDeleteClassroom = actions.deleteClassroom.bind(this, classroomId);
     const students = classroom ? classroom.relationships.students.data : undefined;
     const studentIds = students ? students.map(student => student.id) : undefined;
     const boundDeleteStudent = actions.deleteStudent.bind(this);
+    const classroomAssignments = assignments;
```

Comment:
Ah - then we're doing it wrong. We can subscribe to _part_ of the state object by modifying the `mapStateToProps` function below